### PR TITLE
Adjust web concurrency for management-api and amun-api

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/thoth-application/pull/992

Let's see if the linked issue fixes the problem.

## This introduces a breaking change

- [x] No

### CODE REVIEW

File: amun/base/deploymentconfig.yaml [hunk 1]
```diff
@@ -27,6 +27,8 @@ spec:
               value: "amun.entrypoint:app"
             - name: KUBERNETES_API_URL
               value: "https://kubernetes.default.svc.cluster.local"
+            - name: WEB_CONCURRENCY
+￼              value: "2"
```
Comment (goern) [inline amun/base/deploymentconfig.yaml:31]: a space too much?!

**Review by goern — CHANGES_REQUESTED** [review]

**Comment by fridex** [comment]
PTAL ready to be merged.

**Comment by goern** [comment]
@fridex can we unhold this?

**Pull request merged**